src/recipes/cappuccino.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Cappuccino",
+    "description": "This recipe is for an authentic Italian cappuccino made with the perfect proportions of espresso, steamed milk, and foamed milk. (And by “perfect,” we mean approximately equal proportions of 1/3 each.) For a larger—or more typical American—cappuccino, double the quantities. For a milkier, or wet, cappuccino, increase the milk to 6 tablespoons. Ever wondered the difference between a cappuccino and a macchiato? The latter has just a small spot of foamed milk, as “macchiato” means “spot” in Italian.",
+    "github": "Brand00wn",
+    "ingredients": [
+        {
+            "quantity": "1",
+            "measure": "",
+            "ingredient": "rounded tablespoon finely ground coffee, preferably an espresso blend"
+        },
+        {
+            "quantity": "1/4",
+            "measure": "cup",
+            "ingredient": "cold milk (whole, 2%, or nonfat)"
+        },
+        {
+            "quantity": "1",
+            "measure": "",
+            "ingredient": "pinch cocoa powder or ground cinnamon, optional"
+        }
+    ],
+    "directions": [
+        "Warm the espresso machine. Put the coffee in the filter basket of the portafilter, gently but firmly tamp it down with a tamper, and secure the portafilter in the brew head. Place a 5- or 6-ounce (148- or 177-ml) cup directly under the brew head.",
+        "Brew just long enough to yield 1 to 1 1/2 ounces (30 to 44 ml) of espresso. Turn your machine off just as it finishes pouring the last of the pale-colored crema and before the espresso becomes watery, which you’ll learn with practice. The exact time will depend on your machine.",
+        "Much will depend on your particular machine. Turn on the steamer and let it warm up. Meanwhile, pour the milk into a stainless steel pitcher. Submerge the tip of the steam wand at least half an inch below the milk surface and turn on the steam switch. As the foam rises, gradually lower the pitcher so the tip of the wand remains just below the milk surface. As the milk begins to heat up, tilt the pitcher slightly to swirl the milk and continue to steam until the side of the pitcher becomes almost too hot to hold. Turn off the steam while still holding the tip of the wand under the milk surface.",
+        "Very slowly pour the milk into the cappuccino in one of two ways:
+
+-For a smooth cappuccino in which both the warmed and foamed milk on top is evenly integrated throughout the beverage, gently pour the milk over the espresso in a thin, steady stream.
+-For a cappuccino with a foamy head, hold back the foam with a spoon and gently pour the milk over the espresso. Spoon the foam over the top.",
```

Comment:
Can these be separated into their own items? Like a) and b). This won't pass as valid json.

Reply:
Ohh, fine. I'll fix here, thx!